sway/ipc-server.c

```diff
@@ -419,7 +419,7 @@ void ipc_client_handle_command(struct ipc_client *client) {
 			}
 			json_object_object_add(json, "status_command", json_object_new_string(bar->status_command));
 			json_object_object_add(json, "font", json_object_new_string(bar->font));
-			json_object_object_add(json, "bar_height", json_object_new_int(bar->bar_height));
+			json_object_object_add(json, "height", json_object_new_int(bar->height));
```

Comment:
You can't change this JSON property name - it's backwards compatible with i3bar.